### ISSUE TITLE
Search Add Patient - Missing Where statement

### ISSUE THREAD

**Issue by tmccormi** [LibreEHR/LibreEHR#87]
On any attempted search I get this error:  Notice Nothing between WHERE and ORDER

```
ERROR: query failed: SELECT *, DATE_FORMAT(DOB,'%m/%d/%Y') as DOB_TS FROM patient_data WHERE ORDER BY lname ASC, fname ASC LIMIT 0, 100

Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ORDER BY lname ASC, fname ASC LIMIT 0, 100' at line 1

```

**Comment by teryhill** [comment]
I got a fix for it

**Comment by tmccormi** [comment]
Share, please :-)

**Comment by teryhill** [comment]
be patient 8^)

**Comment by teryhill** [comment]
Here you go

**Comment by aethelwulffe** [comment]
Status update or timeline please?

**Comment by tmccormi** [comment]
@kchapple - I know you are swamped.  Can you point me to this fix?  Is it in the BPUC repo?

**Comment by aethelwulffe** [comment]
if ( $where ) {
    $where .= " AND ";
  }
  $where .= do_action( 'filter_patient_select', $_SESSION['authUser'] );
...is how it is in the suncoastconnection/MI2BPUC/master repo

**Comment by aethelwulffe** [comment]
Not causing any issue there.

**Comment by tmccormi** [comment]
@kchapple Did this get fixed?

**Comment by aethelwulffe** [comment]
Got issue now:
Input array has 2 params, does not match query: 'SELECT *, DATE_FORMAT(DOB,'%m/%d/%Y') as DOB_TS FROM patient_data WHERE ORDER BY lname ASC, fname ASC LIMIT 0, 100'

ERROR: query failed: SELECT *, DATE_FORMAT(DOB,'%m/%d/%Y') as DOB_TS FROM patient_data WHERE ORDER BY lname ASC, fname ASC LIMIT 0, 100


C:\xampp\htdocs\LibreEHR\interface\main\finder\patient_select.php at 204:sqlStatement

**Comment by aethelwulffe** [comment]
do_action ain't doin'

**Comment by kchapple** [comment]
Got it, new/search patient. Will have a fix shortly.

**Comment by aethelwulffe** [comment]
merged commit not referenced, but merged #187
